extension/otto/gohan_core.go

```diff
@@ -82,6 +83,26 @@ func init() {
 				value, _ := vm.ToValue(response)
 				return value
 			},
+			"gohan_trigger_event": func(call otto.FunctionCall) otto.Value {
```

Comment:
It is not used in golang extension but is used in the JS example of GO <-> JS extensions events propagation.